data/mods/PK_Rebalance v0.75/pk_critter_zombie.json

```diff
@@ -0,0 +1,939 @@
+      "half_life": 35
+      },
+    "hp":115,
+    "death_function": "NORMAL",
+    "flags":["SEES", "HEARS", "REVIVES", "SWIMS", "STUMBLES", "SMELLS", "WARM", "BASHES", "GRABS", "GROUP_BASH", "POISON", "NO_BREATHE"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_master",
+    "edit-mode" : "modify",
+        "upgrades":{
+      "into": "mon_zombie_lord",
+      "half_life": 35
+      },
+      "death_function":["NORMAL", "BLOBSPLIT"],
+      "special_attacks":[["UPGRADE", 20],  ["FORMBLOB", 300]]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_necro",
+    "edit-mode" : "modify",
+        "upgrades":{
+      "into": "mon_zombie_lord",
+      "half_life": 42
+      },
+      "death_function":["NORMAL", "BLOBSPLIT"],
+      "special_attacks":[["RESURRECT", 20],  ["FORMBLOB", 300]]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_hulk",
+    "edit-mode" : "modify",
+        "upgrades":{
+      "into": "mon_zombie_hulk_pk",
+      "half_life": 42
+      }
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_brute_ninja",
+    "name": "zombie nightstalker",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"dkgray_red",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":24,
+    "aggression":100,
+    "morale":100,
+    "speed":110,
+    "melee_skill":10,
+    "melee_dice":6,
+    "melee_dice_sides":4,
+    "melee_cut":6,
+    "dodge":5,
+    "armor_bash":14,
+    "armor_cut":8,
+    "armor_stab":10,
+    "vision_day":40,
+    "vision_night":20,
+    "death_drops": "default_zombie_death_drops",
+    "hp":220,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SMASH", 30], ["STRETCH_ATTACK", 3], ["LONGSWIPE", 15]],
+    "description":"Somehow this brute hides in the dark like some kind of boogeyman, where its massive hands flex gnarled claws.  Very agile for such a large zombie.",
+    "flags":["SEES", "HEARS", "GOODHEARING", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON", "NIGHT_INVISIBILITY"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_brute_grappler",
+    "name": "zombie wrestling brute",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"red_white",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":24,
+    "aggression":100,
+    "morale":100,
+    "speed":95,
+    "melee_skill":7,
+    "melee_dice":3,
+    "melee_dice_sides":10,
+    "melee_cut":1,
+    "dodge":3,
+    "armor_bash":10,
+    "armor_cut":14,
+    "armor_stab":8,
+    "vision_day":40,
+    "vision_night":3,
+    "death_drops": "default_zombie_death_drops",
+    "hp":280,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SMASH", 30], ["BIO_OP_TAKEDOWN", 20], ["RANGED_PULL", 20], ["GRAB_DRAG", 10],
+            {
+            "type" : "bite",
+            "cooldown" : 10,
+            "no_infection_chance" : 20,
+            "min_mul" : 0.5, "//" : "stronger jaws"

```

Comment:
At this point in evolution, regular bite is rather weak. You could buff it by copying the damage structures from (vanilla) feral hunter.

`no_infection_chance` is negative - the higher it is, the less likely it is to infect.